src/sql/parts/notebook/notebook.component.ts

```diff
@@ -43,7 +43,7 @@ export const NOTEBOOK_SELECTOR: string = 'notebook-component';
 })
 export class NotebookComponent extends AngularDisposable implements OnInit {
 	@ViewChild('toolbar', { read: ElementRef }) private toolbar: ElementRef;
-	private _model: NotebookModel;
+	public _model: NotebookModel;
```

Comment:
Please do not make this public, it's marked as `_` which indicates private fields. If you need it, consider adding a getter. But I'd question what is accessing this rather than having it passed in?

Reply:
This shouldn't have been done. My bad. Thanks for pointing.